Makefile

```diff
@@ -8,8 +8,7 @@ clean:
 		rm -fr *.egg-info
 
 install:
-		pip install -e .
-		pip install -r dev_requirements.txt
+		pip install -e .[test]
```

Comment:
Ahh, yes, we should quote that entire pip argument. I thought that you meant that we should quote `pip install -e .["test"]`.